vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/gridProConnector.js

```diff
@@ -1,39 +1,45 @@
-window.Vaadin.Flow.gridProConnector = {
-  setEditModeRenderer: function(column, component) {
-      column.editModeRenderer = function(root) {
-          root.appendChild(component);
-          this._grid._cancelStopEdit();
-          component.focus();
-      };
+(function () {
+    const tryCatchWrapper = function (callback) {
+        return window.Vaadin.Flow.tryCatchWrapper(callback, 'Vaadin Grid Pro', 'vaadin-grid-pro-flow');
+    };
 
-      column._setEditorValue = function(editor, value) {
-          // Not needed in case of custom editor as value is set on server-side.
-          // Overridden in order to avoid blinking of the cell content.
-      }
+    window.Vaadin.Flow.gridProConnector = {
+        setEditModeRenderer: tryCatchWrapper(function (column, component) {
+            column.editModeRenderer = tryCatchWrapper(function (root) {
+                root.appendChild(component);
+                this._grid._cancelStopEdit();
+                component.focus();
+            });
 
-      column._getEditorValue = function(editor) {
-          // Not needed in case of custom editor as value is set on server-side.
-          // Overridden in order to avoid blinking of the cell content.
-          return;
-      }
-  },
+            column._setEditorValue = tryCatchWrapper(function (editor, value) {
+                // Not needed in case of custom editor as value is set on server-side.
+                // Overridden in order to avoid blinking of the cell content.
+            })
 
-  patchEditModeRenderer: function(column) {
-      column.__editModeRenderer = function(root, column, rowData) {
-          const cell = root.assignedSlot.parentNode;
-          const grid = column._grid;
+            column._getEditorValue = tryCatchWrapper(function (editor) {
+                // Not needed in case of custom editor as value is set on server-side.
+                // Overridden in order to avoid blinking of the cell content.
+                return;
+            })
```

Comment:
These assignments (`_setEditorValue` and `_getEditorValue`) should end in `;`.

But also the `tryCatchWrapper` is unnecessary here, since these functions don't do anything.

But while changing this part, might as well shorten it from the current:
```js
            column._setEditorValue = tryCatchWrapper(function (editor, value) {
                // Not needed in case of custom editor as value is set on server-side.
                // Overridden in order to avoid blinking of the cell content.
            })

            column._getEditorValue = tryCatchWrapper(function (editor) {
                // Not needed in case of custom editor as value is set on server-side.
                // Overridden in order to avoid blinking of the cell content.
                return;
            })
```
to something like:
```js
            // These are not needed in case of custom editor as value is set on server-side.
            // Overridden in order to avoid blinking of the cell content.
            column._setEditorValue = () => {};
            column._getEditorValue = () => {};
```

Reply:
Updated, but not using arrow functions.

Reply:
Thanks, it's better now.

Imo it's unnecessary to list the parameters for these functions since they are unused here and especially since these are deliberately no-op functions in this case.

I think it looks a bit weird that one has an explicit `return;` though both functions do exactly the same (they both return `undefined`). But I can kind of understand having the explicit return statement on the getter.

I'd still prefer my version above (with empty arrow functions), but it's ok. Will approve as is.